src/components/UI/EmojiPicker/EmojiPicker.tsx

```diff
@@ -1,15 +1,30 @@
 import data from '@emoji-mart/data';
 import Picker from '@emoji-mart/react';
-
+import { useLexicalComposerContext } from '@lexical/react/LexicalComposerContext';
+import { $createTextNode, $getSelection, $isRangeSelection } from 'lexical';
 export interface EmojiPickerProps {
   displayStyle: object;
   onEmojiSelect: Function;
 }
 
-export const EmojiPicker = ({ displayStyle, onEmojiSelect }: EmojiPickerProps) => (
-  <div data-testid="emoji-container" style={displayStyle}>
-    <Picker data={data} onEmojiSelect={onEmojiSelect} />
-  </div>
-);
+export const EmojiPicker = ({ displayStyle, onEmojiSelect }: EmojiPickerProps) => {
```

Comment:
```suggestion
export const EmojiPicker = ({ displayStyle }: EmojiPickerProps) => {
```
Remove this if not needed. 
Also I am not sure if we should tightly couple editor with the picker. There could be a case where the emoji picker is needed without the editor. Let's discuss this once